src/components/FileUpload/FileUpload.jsx

```diff
@@ -20,80 +20,108 @@ registerPlugin(
-    return { abort };
-  };
+      // TODO - in order to handle abort, we need to pass an AbortController to SwaggerRequest, implement this as a future story (it's not working as-is)
+      // https://github.com/swagger-api/swagger-js/blob/master/docs/usage/http-client.md#browser
+      return { abort };
+    };
 
-  const revertFile = (uploadId, load, error) => {
-    deleteUpload(uploadId)
-      .then(() => {
-        load();
-        handleOnChange();
-      })
-      .catch(error);
-  };
+    const revertFile = (uploadId, load, error) => {
+      deleteUpload(uploadId)
+        .then(() => {
+          load();
+          handleOnChange();
+        })
+        .catch(error);
+    };
 
-  const handleProcessFile = () => {
-    handleOnChange();
-  };
+    const handleProcessFile = () => {
+      handleOnChange();
+    };
 
-  const serverConfig = {
-    url: '/internal',
-    process: processFile,
-    fetch: null,
-    revert: revertFile,
-  };
+    const serverConfig = {
+      url: '/internal',
+      process: processFile,
+      fetch: null,
+      revert: revertFile,
+    };
 
-  /**
-   * Default FilePond instance props
-   * If these need to be overwritten, they can be exposed as a prop on this
-   * component and passed through (like labelIdle)
-   */
-  const filePondProps = {
-    allowMultiple: true,
-    server: serverConfig,
-    imagePreviewMaxHeight: 100,
-    labelIdle: isMobile() ? labelIdleMobile : labelIdle,
-    acceptedFileTypes: ['image/jpeg', 'image/png', 'application/pdf'],
-    maxFileSize: '25MB',
-    credits: false,
-  };
+    /**
+     * Default FilePond instance props
+     * If these need to be overwritten, they can be exposed as a prop on this
+     * component and passed through (like labelIdle)
+     */
+    const filePondProps = {
+      server: serverConfig,
+      imagePreviewMaxHeight: 100,
+      labelIdle: isMobile() ? labelIdleMobile : labelIdle,
+      maxFileSize: '25MB',
+      credits: false,
+    };
 
-  /* eslint-disable react/jsx-props-no-spreading */
-  return (
-    <FilePond ref={ref} {...filePondProps} name={name} onprocessfile={handleProcessFile} onaddfilestart={onAddFile} />
-  );
-  /* eslint-enable react/jsx-props-no-spreading */
-});
+    /* eslint-disable react/jsx-props-no-spreading */
+    return (
+      <FilePond
+        ref={ref}
+        {...filePondProps}
+        allowMultiple={allowMultiple}
+        maxParralelUploads={maxParralelUploads}
+        acceptedFileTypes={acceptedFileTypes}
+        name={name}
+        onprocessfile={handleProcessFile}
+        onaddfilestart={onAddFile}
+      />
+    );
+    /* eslint-enable react/jsx-props-no-spreading */
+  },
+);
 
 FileUpload.propTypes = {
-  name: PropTypes.string,
-  createUpload: PropTypes.func,
-  onChange: PropTypes.func,
-  onAddFile: PropTypes.func,
+  name: string,
+  createUpload: func,
+  onChange: func,
+  onAddFile: func,
+  acceptedFileTypes: arrayOf(string),
+  allowMultiple: bool,
+  maxParralelUploads: int,
```

Comment:
I think this should be number. We would also need to update the import
```suggestion
  maxParralelUploads: number,
```
```suggestion
  maxParralelUploads: int,
```